.minedonrc.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "Jeff Mod X",
+  "author": {
+    "name": "LinkWarrior75",
+    "link": "https://github.com/LinkWarrior75"
+  },
+  "icon": "BP/pack_icon.png"
```

oops forgot a comma there

```suggestion
  "icon": "BP/pack_icon.png",
```